lib/content-services/src/lib/aspect-list/aspect-list.component.spec.ts

```diff
@@ -250,6 +250,21 @@ describe('AspectListComponent', () => {
                 component.clear();
                 expect(component.nodeAspects.length).toBe(0);
             });
+
+            it('should store node aspects that are not listed and emit then on value change', async () => {
+                const storedAspect: string[] = ['stored:aspect'];
```

Comment:
no need to set type in this case because ts sees that you assign array with string so string[] type will be set automatically